app/src/middleware/validation.js

```diff
@@ -0,0 +1,27 @@
+const config = require('config');
+const Problem = require('api-problem');
+const { validators }= require('../components/validators');
```

Comment:
Make sure to use VSCode's format document function `Shift + Alt + F` to keep the code spacing and style consistent with whatever is defined in eslint.